extensions/amp-wordpress-embed/1.0/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Oh, one thing: please change `Bento Wordpress Embed` to `Bento WordPress Embed`

Reply:
Done